deltachat-ffi/deltachat.h

```diff
@@ -3576,6 +3576,17 @@ char*           dc_msg_get_setupcodebegin     (const dc_msg_t* msg);
  */
 char* dc_msg_get_videochat_url (const dc_msg_t* msg);
 
+/**
+ * Get the error of the message.
+ * If there is no error associated with the message, an empty string is returned.
+ * NULL is never returned.
```

Comment:
I would rather have it return NULL then allocate an empty string for each message just to notify the UI that there is no error.